src/tests/file-api-test/file-api-test.c

```diff
@@ -173,10 +173,9 @@ void TestCreatRemove(void)
     UtAssert_True(status == OS_SUCCESS, "status after remove max name length file = %d", (int)status);
 
     /* try creating with file name too big, should fail */
-    fd     = ~OS_OBJECT_ID_UNDEFINED;
     status = OS_OpenCreate(&fd, longfilename, OS_FILE_FLAG_CREATE | OS_FILE_FLAG_TRUNCATE, OS_READ_WRITE);
     UtAssert_True(status < OS_SUCCESS, "status after create file name too long = %d", (int)status);
-    UtAssert_UINT32_EQ(fd, OS_OBJECT_ID_UNDEFINED);
+    UtAssert_True(!OS_ObjectIdDefined(fd), "fd == OS_OBJECT_ID_UNDEFINED");
```

Comment:
Can this print out the actual value of fd instead of just always printing that it equals Undefined.

Reply:
Updated in 56cf1ec